website/registrations/models.py

```diff
@@ -275,6 +277,38 @@ def language(self):
         verbose_name=_("birthday calendar opt-in"), default=False
     )
 
+    # ---- Bank account -----
+
+    direct_debit = models.BooleanField(
+        null=False,
+        blank=False,
+        default=False,
+        help_text="If selected, when the registration is accepted, immediately a "
+        "Thalia Pay payment will be created for this user and the registration "
+        "will be completed directly. This can only be selected if a bank "
+        "account is added with direct debit authorization.",
```

Comment:
```suggestion
        help_text=_("When the registration is accepted and this checkbox is enabled, a "
        "Thalia Pay payment will be created for this user and the registration "
        "will be completed immediately. This can only be selected if a bank "
        "account is added with direct debit authorisation during registration."),
```